README.md

```diff
@@ -1 +1,76 @@
-# hello_microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello Microverse
+
+> Description the project.
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Added descriptive README file my project.
+Add 3 files each one in a separate commit:
+A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+A .css file that includes one class with styles for your header (make it in your favorite color).
+A test.md file with the text "This file should be ignored by git".
+
+## Built With
+HTML AND CSS
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [OPTIONAL] Please make sure that your getting started section is more descriptive and the content should be the steps to follow to get your project up and running on a local machine. 

For example,
```
You can add the steps to follow to clone your project,

Step 1:if you dont have git installed on your machine you cad download it from [here](https://git-scm.com/downloads).

Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine


`git clone <your project link>`

Alternatively you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`

Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
```